packages/core/src/components/ui/ProductGallery/ProductGallery.tsx

```diff
@@ -67,6 +68,7 @@ export interface ProductGalleryProps {
   }
   sortBySelector?: SortProps
   productCard?: Pick<ProductCardProps, 'showDiscountBadge' | 'bordered'>
+  emptyGallery?: EmptyGalleryProps
```

Comment:
hmm, we don't use this prop here in the `ui/ProductGallery`. How about remove this line and its import and use it directly in https://github.com/AilinMacchia/faststore/blob/feature/empty-gallery-override/packages/core/src/components/sections/ProductGallery/ProductGallery.tsx#L28C27-L28C27 ?